typescript/agentkit/src/action-providers/vaultsfyi/schemas.ts

```diff
@@ -0,0 +1,109 @@
+import { z } from "zod";
+
+/**
+ * Action schemas for the vaultsfyi action provider.
+ *
+ * This file contains the Zod schemas that define the shape and validation
+ * rules for action parameters in the vaultsfyi action provider.
+ */
+
+const networkSchema = z.enum([
+  "mainnet",
+  "arbitrum",
+  "optimism",
+  "polygon",
+  "base",
+  "gnosis",
+  "unichain",
+]);
+
+/**
+ * Vaults list action schema.
+ */
+export const vaultsActionSchema = z.object({
+  /**
+   * Optional: Name or symbol of the token to filter vaults by
+   */
+  token: z.string().optional(),
```

Comment:
Please add `.describe()` to the schemas to define each variable,
e.g. `token: z.string().optional().describe("Name of symbol of the token to filter vaults by")`

Assume that the describe both replaces the need for a comment AND might be used by the AI one day to infer variable usage. Have both use cases in mind, and write concise short describes.

You may delete these comments after adding in the describes as well